### PR TITLE
feat(tsri, *): delete preferred name as a feature and hide flair for tsri

### PR DESCRIPTION
**Pull request by Itrulia**

### CODE REVIEW

File: apps/tsri/pages/profile/index.tsx [hunk 1]
```diff
@@ -43,7 +43,10 @@ function Profile() {
         </SubscriptionLink>
       </ProfileHeading>
 
-      <PersonalDataFormContainer mediaEmail="info@tsri.ch" />
+      <PersonalDataFormContainer
+        mediaEmail="info@tsri.ch"
+        fields={['firstName', 'flair', 'address', 'password', 'image']}
```
Comment (Itrulia) [inline apps/tsri/pages/profile/index.tsx:48]:
```suggestion
        fields={['firstName', 'address', 'password', 'image']}
```

File: libs/api/prisma/migrations/20240909161100_delete_preferred_name/migration.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `preferredName` on the `users` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "users" DROP COLUMN "preferredName";
```
Comment (michael-scheurer) [inline libs/api/prisma/migrations/20240909161100_delete_preferred_name/migration.sql:8]: I will double-check with tsri.

File: apps/tsri/pages/profile/index.tsx [hunk 3]
```diff
@@ -43,7 +43,10 @@ function Profile() {
         </SubscriptionLink>
       </ProfileHeading>
 
-      <PersonalDataFormContainer mediaEmail="info@tsri.ch" />
+      <PersonalDataFormContainer
+        mediaEmail="info@tsri.ch"
+        fields={['firstName', 'address', 'password', 'image']}
```
Comment (michael-scheurer) [inline apps/tsri/pages/profile/index.tsx:48]: Missing _name, zip, city, country, email_ compared to now

image

**Review by michael-scheurer — CHANGES_REQUESTED** [review]

File: libs/api/prisma/migrations/20240909161100_delete_preferred_name/migration.sql (hunk 2 above)
Comment (Itrulia) [reply libs/api/prisma/migrations/20240909161100_delete_preferred_name/migration.sql:8]: This story was about Tsri not wanting this

File: apps/tsri/pages/profile/index.tsx (hunk 3 above)
Comment (Itrulia) [reply apps/tsri/pages/profile/index.tsx:48]: image
image

did you check the wrong forms?

**Review by michael-scheurer — APPROVED** [review]

**Pull request merged**